doc-surrealdb_versioned_docs/version-1.x/faqs/known-issues.mdx

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_position: 2
+sidebar_label: Known Issues
+title: Known Issues
+description: The purpose of this document is to hightlight some of the known issues with SurrealDB. This will help you to understand the current limitations and issues that are being worked on.
+---
+
+# Known Issues
+
+Here are some known bugs and issues with SurrealDB:
+
+## SDKs
+
+- Some query functionality is not yet available in the SDKs. Such as `IF EXISTS` and `IF NOT EXISTS` clauses in the [DEFINE statement](/docs/surrealdb/surrealql/statements/define/). This means that if you used these clauses in your query method(as seen in the example below), the SDK would not be able to parse the query correctly.
+
+```js
+await db.query(
+    'DEFINE FIELD IF NOT EXISTS email ON TABLE user TYPE string;'
+);
+```
+
+## SurrealQL
+
+- In the [`DELETE` statement](/docs/surrealdb/surrealql/statements/delete) the `ONLY` clause doesn't work as expected because it checks for a length of 1 and DELETE doesn't return anything by default at this time. So it always gives an error.
```

Comment:
```suggestion
- In the [`DELETE` statement](/docs/surrealdb/surrealql/statements/delete) the `ONLY` clause doesn't work as expected because it checks for a length of 1 and DELETE doesn't return anything by default at this time. So it always gives an error. However, adding RETURN $before will force DELETE to return the deleted record, which will be the expected length of 1.

```surql
CREATE person:one;

// Currently fails
DELETE ONLY person:one;

// Workaround: add RETURN $before
DELETE ONLY person:one RETURN $before;
```
```